CHANGELOG.md

```diff
@@ -0,0 +1,84 @@
+# Changelog
+
+<!-- 
+Generate the changelog using the lerna-changelog tool. 
+Run the following command to generate changelog content for pull requests that were merged since the time when the last git tag was created.
+
+./node_modules/lerna-changelog/bin/cli.js --ignoreCommiters
+
+You can also generate changelog content corresponding to pull requests that were merged in between particular git tags. 
```

Comment:
Same here. Can you finish with the `:` as well? or add sth like  `Run this command:` after the existing sentence.